WeRadiate-ThermoSense.ino

```diff
@@ -289,17 +291,35 @@ void setup_platform(void)
                 flags = 0;
                 }
 
+	/* initialize the FLASH */
+	if (gFlash.begin(&gSPI2, Catena::PIN_SPI2_FLASH_SS))
+		{
+		fFlash = true;
+		gFlash.powerDown();
+		gCatena.SafePrintf("FLASH found, put power down\n");
+		}
+	else
+		{
+		fFlash = false;
+		gFlash.end();
+		gSPI2.end();
+		gCatena.SafePrintf("No FLASH found: check wiring\n");
+		}
+
         /* is it modded? */
         uint32_t modnumber = gCatena.PlatformFlags_GetModNumber(flags);
 
-        //fHasPower1 = false;
+        fHasPower1 = false;
 
         if (modnumber != 0)
```

Comment:
for this sketch, the modnumber is not relevant. It is for WeRadiate and will only be used on their boards. You can assume that there's a one wire, no need to check wiring flags.